### PR TITLE
feat(expansion-panel): outlined appearance

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: components/accordion/src/vwc-accordion-base.ts [hunk 1]
```diff
@@ -1,10 +1,22 @@
 import { LitElement, property } from 'lit-element';
 import type { VWCExpansionPanelBase } from '@vonage/vwc-expansion-panel/vwc-expansion-panel-base.js';
+import type { Layout } from '@vonage/vvd-foundation/constants.js';
+
+type AccordionAppearance = Extract<
+	Layout,
+	Layout.Filled | Layout.Outlined | Layout.Soft
```
Comment (yinonov) [inline components/accordion/src/vwc-accordion-base.ts:7]: so we add filled as well?

Comment (rachelbt) [reply components/accordion/src/vwc-accordion-base.ts:7]: OH no - I need to remove it.

File: components/expansion-panel/src/vwc-expansion-panel-base.ts [hunk 2]
```diff
@@ -1,8 +1,19 @@
...
+	Layout.Outlined
+	>;
 
 export abstract class VWCExpansionPanelBase extends LitElement {
+	@property({ type: String, reflect: true })
+		Layout?: ExpantionPanelLayout;
+
 	@property({ type: Boolean, reflect: true })
+
+
```
Comment (yinonov) [inline components/expansion-panel/src/vwc-expansion-panel-base.ts:16]: they belong together
```suggestion
```

File: components/expansion-panel/src/vwc-expansion-panel-base.ts [hunk 3]
```diff
@@ -1,8 +1,19 @@
 import { LitElement, property } from 'lit-element';
 import { observer } from '@material/mwc-base/observer.js';
+import type { Layout } from '@vonage/vvd-foundation/constants.js';
+
+type ExpantionPanelLayout = Extract<
+	Layout,
+	Layout.Outlined
```
Comment (yinonov) [inline components/expansion-panel/src/vwc-expansion-panel-base.ts:7]: the default is ghost right?
```suggestion
	Layout.Outlined,
	Layout.Ghost
```

File: components/expansion-panel/src/vwc-expansion-panel.scss [hunk 4]
```diff
@@ -64,11 +64,26 @@
 			margin-inline-start: 16px;
```
Comment (yinonov) [inline components/expansion-panel/src/vwc-expansion-panel.scss:64]: any style related to the ghost appearance should be selected by
```css
:host(:not([appearance='outlined'i]))
```
as it's the default behaviour, we want to apply its style as a fallback and not by a declarative approach

**Review by yinonov — CHANGES_REQUESTED** [review]

File: components/expansion-panel/src/vwc-expansion-panel.scss (hunk 4 above)
Comment (rachelbt) [reply components/expansion-panel/src/vwc-expansion-panel.scss:64]: I disagree. All the ghost styles are needed in the outlined option. 
The outlined is based in the ghost but has additional style. 
No ghost properties are overide in the outlined style. It seems right to target the outline for that styles.

Comment (yinonov) [reply components/expansion-panel/src/vwc-expansion-panel.scss:64]: Rephrase: any ghost-only styles.
Of course common styles stay intact

**Comment by yinonov** [comment]
@rachelbt 

drop-shadow lacks full parity with box-shadow when it comes to mimicking plain border.
the following drop-shadow mimics exact border behavior
```css
filter: drop-shadow(var(--vvd-color-neutral-30) 1px 0px 0px) drop-shadow(var(--vvd-color-neutral-30) -1px 0px 0px) drop-shadow(var(--vvd-color-neutral-30) 0px -1px 0px) drop-shadow(var(--vvd-color-neutral-30) 0px 1px 0px);
```

along side
```css
contain: content;
```
or
```css
overflow: hidden;
```
it matches the desired style

**Comment by rachelbt** [comment]
If outlined is needed it will be added by wrapping the expansion panel with `<vwc-elevation dp="0">`.
Closing this issue.

**Pull request closed without merging**